data/src/main/java/com/microsoft/azure/kusto/data/Utils.java

```diff
@@ -251,4 +259,28 @@ static String getPackageVersion() {
         }
         return "";
     }
+    static final int SECONDS_PER_MINUTE = 60;
+    static final int MINUTES_PER_HOUR = 60;
+    static final int HOURS_PER_DAY = 24;
+    static final int SECONDS_PER_HOUR = SECONDS_PER_MINUTE * MINUTES_PER_HOUR;
+    static final int SECONDS_PER_DAY = HOURS_PER_DAY * SECONDS_PER_HOUR;
```

Comment:
All of these constants are already defined in the Java time library. See comment above.